lighthouse.ts

```diff
@@ -1,32 +1,25 @@
 import { spawnSync } from 'child_process';
-import sanitize from 'sanitize-filename';
 
 const lighthouseCli = require.resolve('lighthouse/lighthouse-cli');
 
-type OutputFormat = 'json' | 'html' | 'csv';
-
-export const runReport = (url: string, outputFormat: OutputFormat) => {
+// This function is marked as async to wrap it with a promise to make error handling easier
+// Even though the underlying process is spawned synchronously
+// eslint-disable-next-line @cloudfour/typescript-eslint/require-await
+export const runLighthouseReport = async (url: string) => {
   const { status = -1, stdout } = spawnSync('node', [
     lighthouseCli,
     url,
-    `--output=${outputFormat}`,
-    `--output-path=stdout`,
-    `--emulated-form-factor=mobile`,
-    `--only-categories=performance`,
-    `--chrome-flags="--headless"`,
-    `--max-wait-for-load=45000`,
+    '--output=csv',
+    '--output-path=stdout',
+    '--emulated-form-factor=mobile',
+    '--only-categories=performance',
+    '--chrome-flags="--headless"',
+    '--max-wait-for-load=45000',
   ]);
 
   if (status !== 0) {
-    console.error(`Lighthouse report failed for: ${url}`);
-    return false;
+    throw new Error(`Lighthouse report failed for: ${url}`);
```

Comment:
Will this exit the script? Do we catch this somewhere? I believe Lighthouse can fail occasionally even when we do everything right.

Reply:
The error gets caught and handled [here](https://github.com/cloudfour/lighthouse-parade/pull/45/files/8ea2a57832f653daaf288e0b54c640bc5b7cb8b0#diff-d2d9b660b8c342f6126936e111b9d8029438d51672869d23f7a629ce7681fdecR70)